scipy/stats/_morestats.py

```diff
@@ -3910,20 +3910,23 @@ def directionalmean(samples, *, axis=0, normalize=True):
     """
     Computes the directional mean of a sample of vectors.
 
-    Serves as equivalent of the sample mean for directional data whose
-    magnitude is irrelevant, e. g. unit vectors. The vectorial data
-    whose directional mean are computed must be located in the last
-    axis.
+    The directional mean is a measure of "preferred direction" of vector data.
+    It is analogous to the sample mean, but it is for use when the magnitude
+    the data is irrelevant (e.g. unit vectors).
```

Comment:
A few thoughts here:

* Mentioned "preferred direction" to give a sense of what this is for
* Used "analogous" instead of "equivalent" to emphasize that they are different
* Move the part about the last axis to the documentation of the `samples` parameter